<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Главная</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="authButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Авторизоваться</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startTestButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Начать тестирование</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
